test/e2e/e2e_test.go

```diff
@@ -70,7 +70,7 @@ var _ = Describe("e2e", func() {
 			Expect(stdout).To(Equal("active"))
 
 			// Check the podman.socket is running:
-			stdout, err = device.Exec("systemctl --machine flotta@.host is-active --user podman.socket")
```

Comment:
I'm curious, why the change?

Reply:
mainly, because all systemd env vars that are needed, so checking as flotta user using sudo it's much simpler.

Reply:
Strange that the command don't fail on missing env vars. I thought sudo pam modules don't create the needed env vars.